sense/screen/screen.go

```diff
@@ -41,9 +116,10 @@ var displayDevice string
 
 func init() {
 	var err error
+
 	displayDevice, err = getDevice("RPi-Sense FB")
 	if err != nil {
-		panic(err)
+		os.Stdout.WriteString(err.Error() + "\n")
```

Comment:
No, since this makes this package very hard to test. I'm personally of the opinion that a library should never panic, especially not in `init`